xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css

```diff
@@ -167,6 +167,7 @@ thead.xwiki-livetable-display-header td.xwiki-livetable-display-header-filter se
   color: $theme.textPrimaryColor;
   margin: 0;
   padding: 0 0 0 2px;
+  margin-top: 2px;
```

Comment:
Because the margin on `input`s is `0` and doesn't match the bootstrap `btn` and `btn-group` styles.